homeassistant/components/netatmo/sensor.py

```diff
@@ -159,11 +161,8 @@ def find_devices(data):
     """Find all devices."""
     dev = []
     not_handled = []
```

Comment:
this variable is always empty now, is it ok?

Reply:
Well spotted @druidvav.